Source/Core/Core/Boot/Boot.h

```diff
@@ -5,13 +5,16 @@
 #pragma once
 
 #include <cstdlib>
+#include <optional>
 #include <string>
+#include <variant>
 
 #include "Common/CommonTypes.h"
+#include "DiscIO/Enums.h"
+#include "DiscIO/Volume.h"
 
 namespace DiscIO
 {
-class IVolume;
```

Comment:
Looks like it really does not like the forward declaration + putting the default destructor in the implementation file. So I changed it back to including DiscIO/Volume.h and removing the now unnecessary forward declarations.